scripts/cdperf

```diff
@@ -273,13 +273,20 @@ done
 [[ ! $k6_test_host ]] && echo "Error: k6_test_host not set. --cloud-host=https://host.docker.internal:9200" && exit 1
 
 cloud_docker_env=()
+cloud_docker_volumes=()
+cloud_docker_setup_script=""
+cloud_docker_teardown_script=""
 
 if [[ $cloud_vendor == "ocis" ]]
 then
   [[ ! $cloud_docker_image ]] && cloud_docker_image="owncloud/ocis"
   [[ ! $cloud_docker_publish ]] && cloud_docker_publish="9200:9200"
   [[ $cloud_oidc != true ]] && cloud_docker_env+=("PROXY_ENABLE_BASIC_AUTH=true")
+  config_volume_name="ocis-config-k6"
   cloud_docker_env+=("OCIS_INSECURE=true")
+  cloud_docker_volumes+=("$config_volume_name:/etc/ocis")
+  cloud_docker_setup_script="docker run -e IDM_ADMIN_PASSWORD=admin --rm -it -v $config_volume_name:/etc/ocis owncloud/ocis init --insecure true"
```

Comment:
not sure about this, but would it make sense to check for `$config_volume_name/ocis.yml` and only add `cloud_docker_setup_script ` if it does not exist?

Reply:
we could add the check, but it should never exist, because the script deletes the volume in https://github.com/owncloud/cdperf/pull/34/files#diff-3c2eb6391fd7e5b86e8298521670e313ef98296321fb93996d5b558aae2769b7L319
also if it would exist the setup script would silently fail.